### PR TITLE
Clean up VREffect

### PR DESCRIPTION
**Pull request by takahirox**
I did clean up VREffect to let is follow the coding style.

### CODE REVIEW

File: examples/js/effects/VREffect.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ THREE.VREffect = function ( renderer, onError ) {
 	var renderRectL, renderRectR;
 	var eyeFOVL, eyeFOVR;
 
-	function gotVRDisplays( displays ) {
```
Comment (mrdoob) [inline examples/js/effects/VREffect.js]: These are correct actually. There is only space after function:

```js
function () { ... }
```

or

```js
function foo() { ... }
```

Comment (takahirox) [inline examples/js/effects/VREffect.js]: Thanks for the comment.
I'll fix them.

Should we add that to the wiki?
https://github.com/mrdoob/three.js/wiki/Mr.doob%27s-Code-Style%E2%84%A2

**Comment by mrdoob** [comment]
Thanks!

File: examples/js/effects/VREffect.js (hunk 1 above)
Comment (takahirox) [inline examples/js/effects/VREffect.js]: I'm thinking to add this to the wiki.

https://github.com/mrdoob/three.js/wiki/Mr.doob%27s-Code-Style%E2%84%A2

Looks good to you?

## Functions

- The function keyword should always be followed with a space.
- The function name should be followed with an opening parenthesis.

good:

    function ( arg1, arg2 ) {

    }

    function functionName( arg1, arg2 ) {

    }

bad:

    function( arg1, arg2 ) {

    }

    function functionName ( arg1, arg2 ) {

    }

**Pull request merged**